src/Uno.UI.FluentTheme.v1/themeresources_v1.xaml

```diff
@@ -1493,7 +1493,7 @@
       <x:Boolean x:Key="UseSystemFocusVisuals">True</x:Boolean>
       <Thickness x:Key="TextControlBorderThemeThickness">1</Thickness>
       <Thickness x:Key="TextControlBorderThemeThicknessFocused">2</Thickness>
-      <Thickness x:Key="TextControlThemePadding">10,6,6,5</Thickness>
+      <macos:Thickness x:Key="TextControlThemePadding">10,8,6,5</macos:Thickness>
```

Comment:
Should you be adding a `not_macos` as well here ?

Reply:
The tricky part is that I did not modify this file (nor the other one). I updated these two:

```
src/Uno.UI.FluentTheme.v1/Resources/Version1/Priority01/Common_themeresources_v1.xaml
src/Uno.UI.FluentTheme.v2/Resources/Version2/Priority01/Common_themeresources.xaml
```

and when I compiled these two were updated

````
src/Uno.UI.FluentTheme.v1/themeresources_v1.xaml
src/Uno.UI.FluentTheme.v2/themeresources_v2.xaml
````

Should I manually add the values to src/Uno.UI.FluentTheme.v1/themeresources_v1.xaml? I will try and recompile to see how it goes

Reply:
Let's try to make another PR that changes this block: https://github.com/unoplatform/uno/blob/master/src/Uno.UI.FluentTheme.v2/FluentMerge.targets#L10-L39

to use https://github.com/unoplatform/uno.xamlmerge.task instead.